.travis.yml

```diff
@@ -69,10 +70,11 @@ script:
   - ctest -j8 --output-on-failure
 
   ## Run coverage and send result to codecov.io
-  - if [[ "$BTYPE" == "Coverage" ]];
+  - if [[ "$SIMBODY_COVERAGE" == "ON" ]];
     then
-    echo "Coverage";
+    echo "Running coverage target";
     make coverage;
+    echo "Sending coverage result to codecov.io";
     bash <(curl -s https://codecov.io/bash) -X gcov;
     fi
```

Comment:
I've been skeptical about if these multi-line constructs are safe in `travis-ci.yaml`. Do you know more about this? For example, there's some intense discussion here: https://github.com/travis-ci/travis-ci/issues/497. Unless you know that these multi-line constructs won't generate confusing build statuses/error messages, could we put this in one line? Or repeat the `if`, as done elsewhere in the yaml file.